runtime/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OltpExporterHandler.java

```diff
@@ -97,58 +83,17 @@ public OltpExporterHandler(
         this.timeoutInterval = config.timeoutInterval();
         this.warningInterval = config.warningInterval().toMillis();
         this.context = context;
-        OtlpOptionsConfig options = (OtlpOptionsConfig) exporter.options;
-        OtlpEndpointConfig endpoint = options.endpoint;
-        URI location = endpoint.location;
-        OtlpOverridesConfig overrides = endpoint.overrides;
-        this.metricsEndpoint = location.resolve(overrides.metrics);
-        this.logsEndpoint = location.resolve(overrides.logs);
+        this.metricsEndpoint = exporter.resolveMetrics();
```

Comment:
We don't need `metricsEndpoint` or `logsEndpoint` as a field since the endpoint is already in the corresponding client, right?